src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -519,7 +521,6 @@ def insertContainerRules(self):
                     rseName = rseName.replace("cms_type=real", "cms_type=test")
             else:
                 # then it's a T0 container placement
-                ruleKwargs['priority'] = 4
```

Comment:
I guess you are always defining it in the workflow spec and you no longer need it for T0?